CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and the project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Fixed bug where standardizing MolSetGraphs crashed ([#24](https://github.com/microsoft/syntheseus/pull/24)) ([@austint])
```

Comment:
The [changelog format](https://common-changelog.org/#241-change) suggests using imperative mood for the entries (similarly to what's usually done for commit messages).
```suggestion
- Fix bug where standardizing MolSetGraphs crashed ([#24](https://github.com/microsoft/syntheseus/pull/24)) ([@austint])
```